### PR TITLE
Details view for centers

### PR DESCRIPTION
**Pull request by TypicalGitHubUser**
#409 
device-2017-01-02-225519

### CODE REVIEW

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/centerdetails/CenterDetailsFragment.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import butterknife.ButterKnife;
+
+/**
+ * Created by ilya on 12/13/16.
+ */
+
+public class CenterDetailsFragment extends MifosBaseFragment implements CenterDetailsMvpView {
+
+    @BindView(R.id.tv_center_activation_date)
+    TextView activationDate;
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/centerdetails/CenterDetailsFragment.java]: Following [guidelines](https://github.com/openMF/android-client/wiki/Code-style-conventions#51-id-naming) for naming conventions makes it easier for the future developers to work on it.

File: mifosng-android/src/main/res/layout/fragment_center_details.xml [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center" />
+
+    <ScrollView style="@style/ScrollView.Base"
+        android:id="@+id/container">
+
+        <LinearLayout style="@style/LinearLayout.Base">
+
+            <LinearLayout style="@style/LinearLayout.Width">
```
Comment (tarun0) [inline mifosng-android/src/main/res/layout/fragment_center_details.xml]: Redundant nesting of layouts.

File: mifosng-android/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -140,6 +142,10 @@
     <string name="exact_match">Exact Match</string>
     <string name="no_search_result_found">No results found for entered query</string>
     <string name="no_search_query_entered">No Search Query Entered!</string>
+    <string name="active_group_borrowers">Active Group Borrowers</string>
```
Comment (tarun0) [inline mifosng-android/src/main/res/values/strings.xml]: Could have put these in 'Center details related strings'.

**Comment by TypicalGitHubUser** [comment]
@tarun0 Make sense. I did changes.

**Comment by tarun0** [comment]
@TypicalGitHubUser It's failing Travis build.

**Comment by TypicalGitHubUser** [comment]
@tarun0 Fixed error introduced in previous change

File: mifosng-android/src/main/res/layout/fragment_center_details.xml [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+    <ProgressBar
+        android:id="@+id/progress_bar"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center" />
+
+    <ScrollView style="@style/ScrollView.Base"
+        android:id="@+id/container">
+
+        <LinearLayout style="@style/LinearLayout.Base">
```
Comment (tarun0) [inline mifosng-android/src/main/res/layout/fragment_center_details.xml]: @TypicalGitHubUser This layout is still unnecessary.

**Comment by TypicalGitHubUser** [comment]
Fixed

**Comment by therajanmaurya** [comment]
This the feature addition but I am not looking Screenshot. Please include screenshot.

**Comment by therajanmaurya** [comment]
@TypicalGitHubUser Can you please fix this PR and force push again.

**Comment by TypicalGitHubUser** [comment]
@therajanmaurya Fixed

**Comment by therajanmaurya** [comment]
fix in #506

**Comment by therajanmaurya** [comment]
@TypicalGitHubUser good work but need the changes urgent So I have made new PR and took your work in this #506 and enhanced the functionality.

**Pull request closed without merging**